src/viam/sdk/common/utils.cpp

```diff
@@ -55,5 +55,21 @@ std::string bytes_to_string(std::vector<unsigned char> const& b) {
     return img_string;
 };
 
+std::chrono::duration<int64_t, std::micro> from_proto(const google::protobuf::Duration& proto) {
```

Comment:
I think that's some C++ standards pedantry there. There is surely a good reason for it but I don't think it is relevant to our purposes. That's just how many bits is required to meet the needs of `microseconds`, but given that there is no machine I'm aware of with a signed integral type meeting that requirement other than int64_t I think we are safe. If you want to be paranoid, you could always:
```
static_assert(sizeof(std::chrono::microseconds::rep) >= sizeof(std::int64_t));
```
And at least we would fail to compile if our assumptions were wrong.